--- ratpoison-1.4.5/src/history.c~	2009-07-12 23:00:35.000000000 +0200
+++ ratpoison-1.4.5/src/history.c	2009-07-15 16:00:25.000000000 +0200
@@ -75,7 +75,7 @@
 
 #ifndef HAVE_GETLINE
 static ssize_t
-getline(char **lineptr, size_t *n, FILE *f)
+get_line(char **lineptr, size_t *n, FILE *f)
 {
   size_t ofs;
 
@@ -193,7 +193,7 @@
     return;
   }
 
-  while ((linelen = getline (&line, &s, f)) >= 0) {
+  while ((linelen = get_line (&line, &s, f)) >= 0) {
     while (linelen > 0 && (line[linelen-1] == '\n' || line[linelen-1] == '\r')) {
       line[--linelen] = '\0';
     }
